tests/migration_test.go

```diff
@@ -1356,7 +1353,7 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 					// Taint Node.
 					By("Tainting node with kubevirt.io/drain=NoSchedule")
```

Comment:
`kubevirt.io/drain` might not be the taint key anymore, maybe you should change that for better readability in the test log